generator/mysql/query_set.go

```diff
@@ -27,39 +29,77 @@ ORDER BY table_name;
 		return nil, fmt.Errorf("failed to query %s metadata result: %w", tableType, err)
 	}
 
+	tblChan := make(chan int, len(tables))
+	errChan := make(chan error, 1)
+
+	wg := sync.WaitGroup{}
+	for i := 0; i < runtime.NumCPU(); i++ {
```

Comment:
There is no need to limit the number of goroutines to number of CPUs.

Reply:
Removed this and constrained it using 	`db.SetMaxOpenConns(32)` instead.

Reply:
Why is it needed?

Reply:
Aha, I see. We might exceed max number of connections on MySQL. Could you move it into [openConnection](https://github.com/go-jet/jet/blob/e2fe799f8ef4f529b8ca1a660c3b6a494a09dfc7/generator/mysql/mysql_generator.go#L85) so is not called multiple times.